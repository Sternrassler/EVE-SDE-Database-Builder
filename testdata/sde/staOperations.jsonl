{"operationID":1,"activityID":1,"operationName":"Manufacturing","description":"Standard manufacturing operations","fringe":0,"corridor":1,"hub":1,"border":0,"ratio":50}
{"operationID":2,"activityID":3,"operationName":"Research","description":"Research and development operations","fringe":0,"corridor":1,"hub":1,"border":0,"ratio":30}
{"operationID":3,"activityID":1,"operationName":"Trade Hub","description":"Major trading operations","fringe":0,"corridor":0,"hub":1,"border":0,"ratio":100}
