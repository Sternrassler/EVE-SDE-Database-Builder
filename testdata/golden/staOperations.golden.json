[
  {
    "operationID": 1,
    "operationName": "Manufacturing",
    "description": "Standard manufacturing operations",
    "fractionID": null,
    "border": 0,
    "fringe": 0,
    "corridor": 1,
    "hub": 1,
    "ratio": 50,
    "caldariStationTypeID": null,
    "minmatarStationTypeID": null,
    "amarrStationTypeID": null,
    "gallenteStationTypeID": null,
    "joveStationTypeID": null
  },
  {
    "operationID": 2,
    "operationName": "Research",
    "description": "Research and development operations",
    "fractionID": null,
    "border": 0,
    "fringe": 0,
    "corridor": 1,
    "hub": 1,
    "ratio": 30,
    "caldariStationTypeID": null,
    "minmatarStationTypeID": null,
    "amarrStationTypeID": null,
    "gallenteStationTypeID": null,
    "joveStationTypeID": null
  },
  {
    "operationID": 3,
    "operationName": "Trade Hub",
    "description": "Major trading operations",
    "fractionID": null,
    "border": 0,
    "fringe": 0,
    "corridor": 0,
    "hub": 1,
    "ratio": 100,
    "caldariStationTypeID": null,
    "minmatarStationTypeID": null,
    "amarrStationTypeID": null,
    "gallenteStationTypeID": null,
    "joveStationTypeID": null
  }
]